app/assets/stylesheets/earthworks.css

```diff
@@ -2,6 +2,12 @@
   --stanford-50-black: #979694;
 }
 
+/* Selects both #main-content and navbar containers */
+ .container-fluid {
+  max-width:  1600px;
```

Comment:
This seems like you're going to confuse people. A fluid container is defined as not having any max width and no padding.  Are you sure you don't want to add `.container-xxl` instead?

Reply:
Thanks for reviewing Justin. I was not part of any discussions about the ticket parameters. I just know that the design called for a breakpoint at 1600 and 60px of padding.

If adding this CSS to the `container-fluid` class feels like it's going against good practice, maybe we could add an additional class and style that.

Reply:
Something like this: https://github.com/sul-dlss/earthworks/pull/1183

Reply:
Thanks. This looks good but potentially does not align with the designer spec -- I am not sure how strict that is. Let me check.

Reply:
Hi @jcoyne - appreciate that PR example. I just chatted with the workcycle team and learned that we actually do want the behavior of `container-fluid` at all sizes, so even at those smaller breakpoints we want to take up 100% minus 60px.

So I am going to try a different approach than those breakpoints and update this PR.

Reply:
OK, basically it's the same approach, just not targeting `.container-fluid` ...